processing/src/test/java/org/apache/druid/query/aggregation/TestFloatNullableColumnSelector.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation;
+
+
+public class TestFloatNullableColumnSelector extends TestFloatColumnSelector
```

Comment:
nit: is this really worth being in its own file instead of just like.. overriding something inline or a static class in the test file?

Reply:
yeah, I agree this was a quick dirty hack. I remembered this test class a while back: https://github.com/apache/druid/blob/master/processing/src/test/java/org/apache/druid/segment/TestNullableFloatColumnSelector.java. I can cleanup this test to use `TestNullableFloatColumnSelector` which takes in `Float` and determines `isNull()` on the fly as opposed to returning a static value.